Create  "GSoC - the gift of mentorship" blog post (#5770)
jenkins-infra/jenkins.io: content/blog/2022/12/2022-12-09-GSoC-the-gift-of-mentorship.adoc

- events
- developer
authors:
- alyssat
description: >
  A nice gift is to mentor young developers participating in Google Summer of Code (GSoC)
opengraph:
  image: /images/gsoc/opengraph.png
links:
  discourse: https://community.jenkins.io/t/jenkins-in-gsoc-2023-mentors-org-admins-project-ideas-wanted/4387
---

//TODO: image comes here

'Tis the season of gift giving and what better way than to give the gift of mentorship.

Jenkins is a big supporter and contributor of programs like link:https://summerofcode.withgoogle.com/[Google Summer of Code (GSoC)], She Code Africa, and Outreachy. 
We believe in the value and advancements in helping new, and in many cases under-represented contributors to the open source. 
There is also a special 'je ne sais quoi' that comes with giving your time and sharing your knowledge.  

This season of giving, we have opportunities for you to give the gift of mentorship. 
The Jenkins in Google Summer of Code welcome any Jenkins user interested in making a lifelong impact to contributors new to the open source. 

## What does mentorship require?
You know enough about the Jenkins code to guide the GSoC contributor on coding.

## As a mentor, you will:

> - Spend 5-8 hours per week, for 10-22 weeks depending on the size of the project.
> - Provide Jenkins specific code reviews on pull-requests.
> - Review new contributors' proposals, presentations.
> - Complete two (2) evaluation documents from Google.
> - Not work alone, we make sure that every project has at least two mentors.

Reviewer: ```suggestion
* Allocate 5-8 mentoring hours per week, for 10-22 weeks depending on the size of the project.
* Provide Jenkins specific code reviews on pull-requests.
* Review new contributors' proposals and presentations.
* Complete two (2) evaluation documents from Google.
* Collaborate with another mentor on the chosen project, ensuring that no one is working alone.
```
small formatting and wording suggestions, text does **not** need to be updated if this takes away from the message

> ## What will you work on?

Reviewer: ```suggestion
== What will you work on?
```
section header ascii formatting

> - You may choose to be a mentor/co-mentor for any of the link:https://www.jenkins.io/projects/gsoc/2023/project-ideas/[listed project ideas]. 
>   Please note, this list is still evolving.
> - Or link:https://www.jenkins.io/projects/gsoc/proposing-project-ideas/[propose] a project that is near and dear to your heart. 

Reviewer: ```suggestion
* You may choose to be a mentor/co-mentor for any of the link:https://www.jenkins.io/projects/gsoc/2023/project-ideas/[listed project ideas]. 
  Please note, this list is still evolving.
* You can also link:https://www.jenkins.io/projects/gsoc/proposing-project-ideas/[propose] a project that is near and dear to your heart. 
```
small formatting/wording suggestion

> ## Additional info:
> - link:https://developers.google.com/open-source/gsoc/timeline[GSoC 2023 program timeline]
> - link:/projects/gsoc/mentors/[In depth information for mentors]

Reviewer: ```suggestion
== Additional info:
* link:https://developers.google.com/open-source/gsoc/timeline[GSoC 2023 program timeline]
* link:/projects/gsoc/mentors/[In depth information for mentors]
```
small formatting suggestion

> We look forward to welcoming you to this rewarding program. link:https://gitter.im/jenkinsci/gsoc-sig[Reach out to us on the GSoC SIG Gitter chat]. 

Reviewer: ```suggestion
We look forward to welcoming you to this rewarding program! 
link:https://gitter.im/jenkinsci/gsoc-sig[Reach out to us on the GSoC SIG Gitter chat]. 
```

small formatting update

Jenkins in Google Summer of Code welcomes any Jenkins user interested in making a lifelong impact to new contributors new of open source. 

== What does mentorship require?
You know enough about the Jenkins code to guide the GSoC contributor on coding.

== As a mentor, you will:
* Allocate 5-8 mentoring hours per week, for 10-22 weeks depending on the size of the project.
* Provide Jenkins specific code reviews on pull-requests.
* Review new contributors' proposals and presentations.
* Complete two (2) evaluation documents from Google.
* Collaborate with another mentor on the chosen project, ensuring that no one is working alone.

== What will you work on?
* You may choose to be a mentor/co-mentor for any of the link:https://www.jenkins.io/projects/gsoc/2023/project-ideas/[listed project ideas]. 
  Please note, this list is still evolving.
* You can also link:https://www.jenkins.io/projects/gsoc/proposing-project-ideas/[propose] a project that is near and dear to your heart. 

== Additional info:
* link:https://developers.google.com/open-source/gsoc/timeline[GSoC 2023 program timeline]
* link:/projects/gsoc/mentors/[In depth information for mentors]

> We look forward to welcoming you to this rewarding program! 

Reviewer 2: ```suggestion
We look forward to welcoming you as mentors to this rewarding program! 
```
